docker/Dockerfile

```diff
@@ -29,22 +41,35 @@ RUN apt-get update && apt-get install -y \
     rm -rf /var/lib/apt/lists/* && \
     locale-gen en_US.UTF-8
 
-# create non-root user with sudo privileges for certain commands
+# create non-root user with sudo privileges
 RUN groupadd --gid $GID $USER && \
     useradd --uid $UID --gid $GID -m $USER -d /home/${USER} --shell /usr/bin/bash && \
     echo "${USER}:password" | chpasswd && \
     usermod -aG sudo ${USER} && \
     echo "%${USER} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
 
-# add user to dialout group to enable serial port access from the container
-RUN usermod -a -G dialout ${USER}
```

Comment:
Do we really not need this anymore? Do we replace it with anything?

Reply:
this was moved into the leap-specific setup